<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Matricula</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditMatricula">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSearch">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Procurar</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="LabelMovel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Saldo Movel:</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelCartao">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Saldo Cartao:</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelNome">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EditNome">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Comprar:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonSalvar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonSalvar_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioCartao">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cartao</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioMovel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MÃ³vel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
